packages/sparkling-controls/src/ComboBox.svelte

```diff
@@ -32,15 +33,16 @@
     let pathRoot = false
     let pathDrop = false
     let pathItems = false
-    event.path.find((el) => {
+    const path = event.path || (event.composedPath && event.composedPath())
+    path.find((el) => {
       if (el.className === comboRoot.className) pathRoot = true
       if (el.className === comboDrop.className) pathDrop = true
       if (el.className === comboItems.className) pathItems = true
 
       return false
     })
     if (pathRoot && !pathDrop) {
-      if (comboHidden) {
+      if (comboHidden && editable) {
```

Comment:
Could you please clarify a logic, hidden && editable => showCombo?

Reply:
If editable = false it is forbidden to change the selected element